packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md

```diff
@@ -96,7 +96,7 @@ The following commands are supported for the Token Inline Hook type:
 | com.okta.identity.patch | Modify an ID token.     |
 | com.okta.access.patch   | Modify an access token. |
 
-> Note: The `commands` array should only contain commands that can be applied to the requested tokens. For example, if the token is an ID token, the `commands` array should not contain commands of the type `com.okta.access.patch`.
+> Note: The `commands` array should only contain commands that can be applied to the requested tokens. For example, if the token is an ID token, the `commands` array shouldn't contain commands of the type `com.okta.access.patch`.
```

Comment:
How about changing `For example, if the token is an ID token ...` to `For example, if only an ID token token is requested ...`